src/main/resources/update/gitbucket-core_4.12.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<changeSet>
+  <createTable tableName="RELEASE">
+    <column name="USER_NAME" type="varchar(100)" nullable="false"/>
+    <column name="REPOSITORY_NAME" type="varchar(100)" nullable="false"/>
+    <column name="RELEASE_ID" type="int" nullable="false"/>
+    <column name="NAME" type="varchar(100)" nullable="false"/>
+    <column name="TAG" type="varchar(100)" nullable="false"/>
+    <column name="AUTHOR" type="varchar(100)" nullable="false"/>
+    <column name="CONTENT" type="text" nullable="true"/>
+    <column name="IS_DRAFT" type="boolean" nullable="false"/>
+    <column name="IS_PRERELEASE" type="boolean" nullable="false"/>
+    <column name="REGISTERED_DATE" type="datetime" nullable="false"/>
+    <column name="UPDATED_DATE" type="datetime" nullable="false"/>
+  </createTable>
+
+  <addPrimaryKey constraintName="IDX_RELEASE_PK" tableName="RELEASE" columnNames="USER_NAME, REPOSITORY_NAME, RELEASE_ID"/>
+  <addUniqueConstraint constraintName="IDX_RELEASE_UNIQ" tableName="RELEASE" columnNames="USER_NAME, REPOSITORY_NAME, TAG"/>
+  <addForeignKeyConstraint constraintName="IDX_RELEASE_FK0" baseTableName="RELEASE" baseColumnNames="USER_NAME, REPOSITORY_NAME" referencedTableName="REPOSITORY" referencedColumnNames="USER_NAME, REPOSITORY_NAME"/>
+
+  <createTable tableName="RELEASE_ID">
```

Comment:
Why tag name isn't used as release id like GitHub? Even if RELEASE_ID is necessary, why can't AUTO_INCREMENT be used?